README.md

```diff
@@ -37,8 +37,9 @@ Alternatively, you can also refer to them with their fully-qualified name:
 
 ### buffer()
 
-The `buffer(ReadableStreamInterface $stream)` function can be used to create
-a `Promise` which resolves with the stream data buffer.
+The `buffer(ReadableStreamInterface $stream, int $maxLength = 0)` function can be used to create
+a `Promise` which resolves with the stream data buffer. With an optional maximum length argument 
+which defaults to no limit.
```

Comment:
IMO `0` is a valid limit to ensure *no* data is sent. I think a default `null` parameter to mean "no limit" makes more sense here :+1:

Reply:
Changed it, but using a limit of `0` is kinda pointless :zipper_mouth_face:

Reply:
I agree that there aren't many use cases for this, but IMO it makes more sense as a extreme value instead of "no value" which would be closer to the other end of the range :-)

Reply:
True =D